edison/static/js/policy.controller.js

```diff
@@ -10,6 +10,34 @@ function capitalize(string) {
     return `${string.charAt(0).toUpperCase()}${string.slice(1)}`
 }
 
+function commandField(policy)
+{
+    var commandRow = createInitElement('div', 'form-row')
+    var light =   createCommandField('light', policy)
+    var ac =   createCommandField('ac', policy);
+    var shutters =  createCommandField('shutters', policy);
+    
+    commandRow.appendChild(light);
+    commandRow.appendChild(ac);
+    commandRow.appendChild(shutters);
+    return commandRow;
+}
+
+
+function createCommandField(fieldName, policy) {
+
+    var id = `policy-${fieldName}${policy.id}`;
+    var group = createInitElement('div', 'form-group col-md-4');
+    var labelElement = createInitElement('label');
+    var select = createInitElement('select', 'form-control', id);
+    var labelName = capitalize(fieldName);
+
+    labelElement.innerHTML = labelName;
+    group.appendChild(labelElement);
+    group.appendChild(select);
+
+    return group;
+}
 function createTextElementForUpdate(id, fieldName, field) {
 
```

Comment:
i think this space is unnecessary..
@DoRTaL94  just pushed a PR about Lint, maybe you can use the package he used. it called - eslint, it will automatically fix those nonsense

Reply:
I think it's best to wait until the lint CI PR is merged and then merge PRs related to the frontend.
I'll try to make it happen in the next few days.
Because it will be much easier for you to lint your files (I've added scripts for that).

Reply:
Ok, so just do nothing here?

Reply:
It seems like you did. That's okay just please be patient for a couple of days with merging this PR. You can if you'd like send additional PRs related to the frontend but please don't merge them.